linker/Tests/Mono.Linker.Tests.Cases/Reflection/TypeUsedViaReflection.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Diagnostics;
+using System.Reflection;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+
+namespace Mono.Linker.Tests.Cases.Reflection {
+	public class TypeUsedViaReflection {
+		public static void Main ()
+		{
+			TestGetTypeKeptViaReflectionFullString ();
```

Comment:
Please remove `TestGetTypeKeptViaReflection` prefix everywhere, this is very hard to read